src/components/icon/CdrIcon.vue

```diff
@@ -6,64 +6,53 @@
     <slot />
     <use
       v-if="use"
-      v-bind="useAttrs"
+      v-bind="hrefAttrs"
```

Comment:
Renamed this to avoid naming collision with `useAttrs` from the new `<script setup>` syntax.